### PR TITLE
TW-650 Share NFT feature

### PR DESCRIPTION
**Pull request by lendihop**
https://madfish.atlassian.net/jira/software/projects/TW/boards/4?selectedIssue=TW-650

### CODE REVIEW

**Review by lourenc — DISMISSED** [review]
LGTM 👍 Only 1 minor comment

File: src/hooks/use-nft-dynamic-links.hook.ts [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+        if (collectible) {
+          navigate(ModalsEnum.CollectibleModal, { collectible });
+        }
+      } else {
+        showErrorToast({ description: NOT_AUTHORISED_ERROR_MESSAGE });
+      }
+    }
+  };
+
+  const getAppLaunchLink = async () => {
```
[the remark this answers is not in the document]
Comment (lendihop) [reply src/hooks/use-nft-dynamic-links.hook.ts]: renamed

**Review by lourenc — APPROVED** [review]

**Review by lourenc — APPROVED** [review]

File: src/modals/collectible-modal/utils/get-nft-dynamic-url.ts [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+import { DYNAMIC_LINKS_DOMAIN_URI_PREFIX } from 'src/utils/env.utils';
+import { formatImgUri } from 'src/utils/image.utils';
+
+export const getNftDynamicUrl = async (collectible: TokenInterface) =>
+  await dynamicLinks().buildShortLink(
+    {
+      link: `${DYNAMIC_LINKS_DOMAIN_URI_PREFIX}/nft?jsonData=${encodeURIComponent(JSON.stringify(collectible))}`,
+      domainUriPrefix: DYNAMIC_LINKS_DOMAIN_URI_PREFIX,
+      ios: {
+        appStoreId: '1610108763',
```
Comment (svjatoslavtt) [inline src/modals/collectible-modal/utils/get-nft-dynamic-url.ts:13]: is it public data? maybe we need to add this in env

Comment (lendihop) [reply src/modals/collectible-modal/utils/get-nft-dynamic-url.ts:13]: its public. You can take it from app store link https://apps.apple.com/us/app/temple-wallet-mobile/id1610108763

**Review by svjatoslavtt — DISMISSED** [review]

**Review by lourenc — DISMISSED** [review]

**Review by svjatoslavtt — APPROVED** [review]

**Review by lourenc — DISMISSED** [review]

**Review by svjatoslavtt — DISMISSED** [review]

**Review by lourenc — APPROVED** [review]

**Review by svjatoslavtt — APPROVED** [review]

**Review by NovakVlad2424 — APPROVED** [review]
done ios

**Review by vladvn1905 — APPROVED** [review]
Android done

**Pull request closed without merging**